src/core/src/core_logic/PatchInstaller.py

```diff
@@ -72,22 +76,26 @@ def start_installation(self, simulate=False):
 
         # Install Updates
         installed_update_count, update_run_successful, maintenance_window_exceeded = self.install_updates(maintenance_window, package_manager, simulate)
-
+        retry_count = 1
         # Repeat patch installation if flagged as required and time is available
         if not maintenance_window_exceeded and package_manager.get_package_manager_setting(Constants.PACKAGE_MGR_SETTING_REPEAT_PATCH_OPERATION, False):
             self.composite_logger.log("\nInstalled update count (first round): " + str(installed_update_count))
             self.composite_logger.log("\nPatch installation run will be repeated as the package manager recommended it --------------------------------------------->")
             package_manager.set_package_manager_setting(Constants.PACKAGE_MGR_SETTING_REPEAT_PATCH_OPERATION, False)  # Resetting
             new_installed_update_count, update_run_successful, maintenance_window_exceeded = self.install_updates(maintenance_window, package_manager, simulate)
             installed_update_count += new_installed_update_count
+            retry_count = retry_count + 1
 
```

Comment:
L87 shows not covered. 
Suggestions to ensure coverage:

- Use ZypperPackageManager for your test. See Test_ZypperPackageManager.py for reference
- If that doesn't work, set Constants.PACKAGE_MGR_SETTING_REPEAT_PATCH_OPERATION only for this one test and make sure to reset it to it's original value, else it might break other tests

Reply:
The code inside if block is not covered. As discussed, we can do it as a separate task.

Reply:
Make sure you have the follow up task in place @GAURAVRAMRAKHYANI